.env.example

```diff
@@ -52,3 +51,9 @@ MUGSHOT_BINARY_NPM=""
 MUGSHOT_NODE_MODULES_PATH=""
 MUGSHOT_BINARY_CHROME=""
 MUGSHOT_PUPPETEER_PROXY=""
+MUGSHOT_PUPPETEER_DISABLE_SANDBOX=false
```

Comment:
I don't really like negative options. It would be easier to call it `MUGSHOT_PUPPETEER_SANDBOX=true/false`. In the current way you have to think about what a double negative means.

Reply:
fixed ;)